engine/src/main/java/org/hibernate/search/engine/backend/reporting/spi/BackendMappingHints.java

```diff
@@ -20,4 +20,7 @@ public interface BackendMappingHints {
 	@Message(value = "")
 	String noEntityProjectionAvailable();
 
+	@Message( "Add the @ScaledNumberField annotation to this property and configure the `decimalScale` attribute as necessary." )
```

Comment:
That's cheating... `@ScaledNumberField` doens't exist in the engine module either.

This should just be `@Message("")`. The real implementation must be in `StandalonePojoMappingHints`/`HibernateOrmMappingHints`. At best, you can introduce an intermediary interface `PojoBackendMappingHints` to avoid duplicating the same message, but that's about all you can do.